app/src/main/java/com/amaze/filemanager/activities/AboutActivity.java

```diff
@@ -46,15 +46,17 @@
     private int mCount=0;
     private Snackbar snackbar;
     private SharedPreferences mSharedPref;
-    private View mAuthorsDivider;
+    private View mAuthorsDivider, mDeveloper1Divider;
 
     private static final String KEY_PREF_STUDIO = "studio";
     private static final String URL_AUTHOR_1_G_PLUS = "https://plus.google.com/u/0/110424067388738907251/";
     private static final String URL_AUTHOR_1_PAYPAL = "arpitkh96@gmail.com";
     private static final String URL_AUTHOR_2_G_PLUS = "https://plus.google.com/+VishalNehra/";
     private static final String URL_AUTHOR_2_PAYPAL = "https://www.paypal.me/vishalnehra";
     private static final String URL_DEVELOPER1_GITHUB = "https://github.com/EmmanuelMess";
+    private static final String URL_DEVELOPER2_GITHUB = "https://github.com/TranceLove";
     private static final String URL_DEVELOPER1_BITCOIN = "bitcoin:12SRnoDQvDD8aoCy1SVSn6KSdhQFvRf955?amount=0.0005";
+    private static final String URL_DEVELOPER2_DONATE = "https://www.paypal.me/raymondlai";
```

Comment:
Please change to https://paypal.me/TranceLove - I've just set things up there, and I'm using the same profile pic as here.

Reply:
Done